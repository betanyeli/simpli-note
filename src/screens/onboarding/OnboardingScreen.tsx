import { Text, View } from 'react-native'
import React from 'react'
import styles from './OnboardingScreen.styles'

const OnboardingScreen = () => {
    return (
        <View style={styles.container}>
            <Text>OnboardingScreen</Text>
        </View>
    )
}

export default OnboardingScreen

