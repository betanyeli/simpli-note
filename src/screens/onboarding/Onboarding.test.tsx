import OnboardingScreen from "./OnboardingScreen";
import { render } from '@testing-library/react-native';

describe('List Screen Test', () => {
    it('Should match with snapshot', () => {
        const { toJSON } = render(<OnboardingScreen />);
        expect(toJSON()).toMatchSnapshot();
    })
})