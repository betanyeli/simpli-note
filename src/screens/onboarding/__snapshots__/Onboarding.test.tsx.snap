// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`List Screen Test Should match with snapshot 1`] = `
<View
  style={
    {
      "flex": 1,
    }
  }
>
  <Text>
    OnboardingScreen
  </Text>
</View>
`;
